components/apimgt/org.wso2.carbon.apimgt.internal.service/src/main/resources/api.yaml

```
@@ -476,6 +476,39 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  /retrieve-api-artifacts:
+    post:
+      summary: Get all the API runtime artifacts for a given API UUID list and gateway label
```

Summary is too long. Let's make it shorter and add more information to the description
```suggestion
      summary: Get API runtime artifacts from ID list
```

---

Fixed with 87f85e02881b3d64d82f300d60ee009702fb35d5